packages/block-editor/src/components/page-template-picker/default-templates.js

```diff
@@ -1,98 +1,857 @@
 /**
  * WordPress dependencies
  */
-import { parse } from '@wordpress/blocks';
+import { createBlock } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
 
 /**
  * External dependencies
  */
+import { map } from 'lodash';
 import memoize from 'memize';
 
 const defaultTemplates = [
 	{
-		name: 'About',
+		// translators: title for "About" page template
```

Comment:
This file got really big and it may get bigger in the future, could we separate each element of the templates array to its own file?

Reply:
Sounds good! I'll split it 👍

Reply:
Updated =)

Reply:
Thanks a lot